solr/solrj/src/java/org/apache/solr/common/params/QueryElevationParams.java

```diff
@@ -55,4 +55,16 @@
    * they be subject to whatever the sort criteria is?  True by default.
    */
   String USE_CONFIGURED_ELEVATED_ORDER = "useConfiguredElevatedOrder";
-}
\ No newline at end of file
+
+  /**
+   * By default, the component will also elevate docs that aren't part of the search result (matching the query).
+   * If you only want to elevate the docs that are part of the search result, set this to false. True by default.
+   */
+  String ELEVATE_DOCS_WITHOUT_MATCHING_Q = "elevateDocsWithoutMatchingQ";
+
+  /**
+   * If elevation is used in combination with the collapse query parser, we can define that we only want to return the
+   * representative and not all elevated docs.
+   */
+  String ONLY_ELEVATED_REPRESENTATIVE = "onlyElevatedRepresentative";
```

Comment:
Since this is to drive the behavior of the CollapsingQParserPlugin, I think this parameter should be defined in the CollapsingQParserPlugin instead.
For example it could be named COLLECT_ELEVATED_DOCS_WHEN_COLLAPSING (default true).